big-red-shuttle/EmergencyViewController.swift

```diff
@@ -0,0 +1,186 @@
+            
+            return stringOfPhoneNumber
+        } else{
+            
+            return stringOfPhoneNumber
+        }
+    }
+    
+    //MARK: tableview functions
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+
+        return 2
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if section == 1 {
+            return emergencyArray.count
+        }
+        return 1
+    }
+    
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        
+
+        //create everything for first cell
+        if indexPath.section == 0 {
+            let cell = UITableViewCell()
+
+            //imageView
+            let imageView = UIImageView()
+            imageView.image = UIImage(named: "eye")
+            //117 width 63 height
+            imageView.frame = CGRect(x: 0, y: 0, width: 46, height: 32)
+            imageView.center = CGPoint(x: view.frame.midX, y: 40)
+            imageView.tintColor = UIColor(red:0.83, green:0.29, blue:0.24, alpha:1.0)
+            //Safety Label
+            let safetyLabel = UILabel()
+            safetyLabel.text = "Safety is our mission"
+
+            safetyLabel.font = UIFont.boldSystemFont(ofSize: 18)
+            safetyLabel.sizeToFit()
+            safetyLabel.center = CGPoint(x: view.frame.midX, y: 85)
+            
+            //Safety Detail Label
+            let safetyLabelDetail = UILabel(frame: CGRect(x: safetyLabel.frame.minX, y: 130, width: safetyLabel.frame.maxX + 30, height: 10))
+            
+            safetyLabelDetail.textAlignment = .center
+            safetyLabelDetail.text = "If there is an emergency, please call the appropriate service."
+
+            safetyLabelDetail.font = UIFont.systemFont(ofSize: 16)
+            safetyLabelDetail.textColor = .lightGray
+            safetyLabelDetail.numberOfLines = 2
+            safetyLabelDetail.sizeToFit()
+
+            safetyLabelDetail.center = CGPoint(x: view.frame.midX, y: 130)
+
+            cell.addSubview(imageView)
+            cell.addSubview(safetyLabel)
+            cell.addSubview(safetyLabelDetail)
+            
+            cell.isUserInteractionEnabled = false
+            
+            return cell
+        } else {
+            let cell = UITableViewCell(style: .subtitle, reuseIdentifier: "emergencyCell")
+            cell.textLabel?.text = emergencyArray[indexPath.row].0
+            cell.detailTextLabel?.text = phoneNumberToFormattedString(phoneNumber: emergencyArray[indexPath.row].1)
+
+            cell.detailTextLabel?.textColor = .lightGray
+            
+            return cell
+        }
+    }
+    
+    
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        if section == 1 {
+            return "EMERGENCY CONTACTS"
+        }
+        
+        return nil
```

Comment:
Condense lines 127 - 131 to `return (section == 1) ? "EMERGENCY CONTACTS" : nil`